src/test/resources/org/collectionspace/qa/cucumber/supplementary_to_primary_tab/group.feature

```diff
@@ -1,91 +1,121 @@
 # Created by Jennifer Lynn Be on 01/05/2016
 # needs to be tested
 
-@supplementaryprimary
+@supplementaryprimary @group
 Feature: Supplementary Manual QA - Group
 
-  Scenario: New Record Behavior
-    Given user is on the "Create New" page
-      And selects the "Group" radio button on the Create New page
-      And clicks on the Create button
-    Then the "cancel" button "should not" be clickable
-    Then the "delete" button "should not" be clickable
-    When user clicks the "add" button on the "Cataloging" area to the right
-    Then the message "Please save the record you are creating before trying to relate other records to it." should be displayed
-    When user clicks the "add" button on the "Procedures" area to the right
-    Then the message "Please save the record you are creating before trying to relate other records to it." should be displayed
-    Then close the browser
+  # Scenario: New Record Behavior
+  #   Given user is on the "Create New" page
+  #     And selects the "Group" radio button on the Create New page
+  #     And clicks on the Create button
+  #   Then the "cancel" button "should not" be clickable
+  #   Then the "delete" button "should not" be clickable
+  #   When user clicks the "add" button on the "Cataloging" area to the right
+  #   Then the message "Please save the record you are creating before trying to relate other records to it." should be displayed
+  #   When user clicks the "add" button on the "Procedures" area to the right
+  #   Then the message "Please save the record you are creating before trying to relate other records to it." should be displayed
+  #   Then close the browser
 
   Scenario: Record Title
-    Given user is on the "Create New" page
-      And selects the "Group" radio button on the Create New page
-      And clicks on the Create button
+  
+    # Temporary workaround
+    Given user is on a blank "Group" record
+    
+    # Given user is on the "Create New" page
+    ## The following step leaves the Cataloging record's radio button selected,
+    ## rather than selecting the Group record's radio button
+    #   And selects the "Group" radio button
+    #   And clicks on the Create button
+    
       And enters "Blue" in the "Group" "Title" field
-      And adds "Albert Smith" to the "Group" "Group Owner" vocab field
+      And presses the Tab key
```

Comment:
Pressing the Tab key, to tab out of the "Title" field, was necessary to get the title bar to update right away. Otherwise, the title bar updating took more than 10 seconds to occur (at least in my environment), and Cucumber timed out.